irony.el

```diff
@@ -645,8 +645,21 @@ list (and undo information is not kept).")
 (defun irony--run-task-asynchronously (task callback)
   (irony-iotask-schedule (irony--get-server-process-create) task callback))
 
+(defun irony--quote-strings (strings &optional separator)
```

Comment:
Just for future reference, could you comment that the implementation improves over `combine-and-quote-strings`?